usr/sbin/cloudguest-repo-service

```diff
@@ -101,7 +101,11 @@ user, password = utils.get_credentials(
     utils.get_credentials_file(update_server)
 )
 auth_creds = HTTPBasicAuth(user, password)
-instance_data = bytes(utils.get_instance_data(utils.get_config()), 'utf-8')
+instance_data = None
```

Comment:
Inconsistent initialization of instance_data probably good to be `b''` for all instances since `if instance_data` resolves the same for None and empty string.

Reply:
The initialization matches with how the `instance_data` variable is used in the context of this reference. Python, not C.

Reply:
If there isn't an exception they all get initialized by `instance_data = bytes(utils.get_instance_data(utils.get_config()), 'utf-8')` which is bytes?

Reply:
The get_instance_data() function returns a string.